src/vivarium_conic_lsff/data/builder.py

```diff
@@ -210,3 +210,22 @@ def load_and_write_lbwsg_data(artifact: Artifact, location: str):
             data = loader.get_data(key, location)
             logger.debug(f'Writing data for {key} to artifact.')
             write_data_by_draw(artifact, key, data)
+
+
+def load_and_write_vitamin_a_deficiency_data(artifact: Artifact, location: str):
+    keys = [
+        project_globals.VITAMIN_A_DEFICIENCY_CATEGORIES,
+        project_globals.VITAMIN_A_DEFICIENCY_RESTRICTIONS,
+        project_globals.VITAMIN_A_DEFICIENCY_EXPOSURE,
+        project_globals.VITAMIN_A_DEFICIENCY_RELATIVE_RISK,
+        project_globals.VITAMIN_A_DEFICIENCY_PAF,
+        project_globals.VITAMIN_A_SEQ_DW_MODERATE_VI,
```

Comment:
So our model is going to just be susceptible or with condition.  We only need the cause level disability weight, which you should be able to retrieve directly with get_measure

Reply:
also, there's no mortality in this model, yeah?
Maybe write a comment here to that effect.

Reply:
Done and done.